src/code42cli/cmds/devices.py

```diff
@@ -330,13 +342,61 @@ def list_devices(
         df = _add_settings_to_dataframe(state.sdk, df)
     if include_usernames:
         df = _add_usernames_to_device_dataframe(state.sdk, df)
+    if include_legal_hold_membership:
+        df = _add_legal_hold_membership_to_device_dataframe(state.sdk, df)
     if df.empty:
         click.echo("No results found.")
     else:
         formatter = DataFrameOutputFormatter(format)
         formatter.echo_formatted_dataframe(df)
 
 
+def _add_legal_hold_membership_to_device_dataframe(sdk, df):
+    matters = _get_all_active_matters(sdk)
+    matters = [matter["legalHoldUid"] for matter in matters]
+
+    legal_hold_members = []
+
+    for matter in matters:
+        memberships = _get_legal_hold_memberships_for_matter(
+            sdk, matter_id=matter, active=None
```

Comment:
I changed it in my example `_get_all_active_hold_memberships` above, but does it make sense to include inactive memberships in these results? If we only get active memberships, then we can drop the `legalHoldMemberActive ` column from the results and the data will read cleaner.

Reply:
I had included inactive memberships as a relic from when this code lied in the `legal-hold` command, where there is an `--include-inactive` option. I checked with Paul Hirst, and there's not much of a use case for this, so I'll go with your suggestion.